packages/validator/src/services/prepareBeaconProposer.ts

```diff
@@ -29,26 +37,46 @@ export class PrepareBeaconProposerService {
     clock.runEverySlot(this.prepareBeaconProposer);
   }
 
-  private prepareBeaconProposer = async (epoch: Epoch): Promise<void> => {
+  private prepareBeaconProposer = async (slot: Slot): Promise<void> => {
```

Comment:
Do we really need to run this every slot? With high validator counts that a lot of useless signatures.